plutus-core/untyped-plutus-core/src/UntypedPlutusCore/Evaluation/Machine/SteppableCek/Internal.hs

```diff
@@ -454,7 +455,17 @@ evalBuiltinApp fun term runtime = case runtime of
 {-# INLINE evalBuiltinApp #-}
 
 spendBudgetCek :: GivenCekSpender uni fun s => ExBudgetCategory fun -> ExBudget -> CekM uni fun s ()
-spendBudgetCek = let (CekBudgetSpender spend) = ?cekBudgetSpender in spend
+spendBudgetCek = unCekBudgetSpender ?cekBudgetSpender
+
+-- | Spend each budget from the given stream of budgets.
+spendBudgetStreamCek
```

Comment:
I see that you renamed and inlined spendBudgetStreamCek to spendBudgets. I don't like that the names start to diverge so much between the normal cek and steppablecek. Can we at least use the same names? I don't mind though that things moved into enterComputeCek in the normal cek.

Reply:
Fair enough, I've made `evalBuiltinApp` the same in both the machines, which includes removing `spendBudgetStreamCek` from the steppable one.